code/modules/power/supermatter/supermatter.dm

```diff
@@ -178,7 +195,14 @@ GLOBAL_DATUM(main_supermatter_engine, /obj/machinery/power/supermatter_crystal)
 	var/bullet_energy = 2
 	///How much hallucination should we produce per unit of power?
 	var/hallucination_power = 0.1
-
+	///The efficiency of the antinoblium interaction.
+	var/antinoblium_efficiency = 0.0
+	///Delta from antinoblium_efficiency and antinoblium composition.
+	var/antinoblium_efficiency_delta = 0.0
+	///Inverse of 1 - antinoblium_efficiency.
+	var/antinoblium_multiplier = 1.0
+	///Powerloss composition.
```

Comment:
This too

Reply:
The delta is just the difference between efficiency and antinob composition. The multiplier is used to multiply some effects of the SM.